lesson4/argirovatina_hometask.rb

```diff
@@ -11,7 +11,8 @@ def lazy?(key)
   end
 
   def respond_to_missing?(method_name, include_private = false)
-    self.has_key? method_name || super
+    key = get_correct_key(method_name)
```

Comment:
no. because if I ask p hash.respond_to? 'proc_key' it sends key as symbol, although i asked for a string. so i re-check it. other way i get false instead of true. run code with this string commented